packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -762,3 +762,14 @@ class _TestTicker extends Ticker {
     super.dispose();
   }
 }
+
+mixin UpdateCount on RenderObjectWidget {
```

Comment:
I wouldn't add this new mixin to widget_tester. Just checking that the renderObject's property changed as result of pumping a new widget is ok.

Reply:
Done :)